program/convert-string-to-vowelcase/convertStringToVowelCase.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+
+int main()
+{
+  char mystring[30];
+  
+  printf("Please input string for conversion: ");
+
+  scanf("%s", &mystring);
+
+  for (size_t i = 0; i < sizeof(mystring); i++)
+  {
+    if (mystring[i] >=65 && mystring[i] <=90)
+    {
+      mystring[i] += 32;
+    }
+    
+    switch (mystring[i])
+    {
+    case 'a':
+      mystring[i] = 'A';
+      break;
+
+    case 'e':
+      mystring[i] = 'E';
+      break;
+
+    case 'i':
+      mystring[i] = 'I';
+      break;
+
+    case 'o':
+      mystring[i] = 'O';
+      break;
+
+    case 'u':
+      mystring[i] = 'U';
+      break;
+    
+    default:
+      break;
+    }
```

Comment:
This code works only for one word , if we are having more words separated by space , it just returns the first word

Reply:
Work fine

![Screenshot from 2023-01-11 21-52-30](https://user-images.githubusercontent.com/54644599/211860270-4223799f-0027-4859-b05d-934bcae1e958.png)